diff --git a/build/renderRootComponent.js b/build/renderRootComponent.js
index 0b4f220..270f57a 100644
--- a/build/renderRootComponent.js
+++ b/build/renderRootComponent.js
@@ -83,13 +83,13 @@ function renderRootComponent(Component) {
             SplashScreen._internal_preventAutoHideAsync?.();
         });
         React.startTransition(() => {
-            if (process.env.NODE_ENV !== 'production') {
-                const { withErrorOverlay } = require('@expo/metro-runtime/error-overlay');
-                (0, expo_1.registerRootComponent)(withErrorOverlay(Component));
-            }
-            else {
+            // if (process.env.NODE_ENV !== 'production') {
+            //     const { withErrorOverlay } = require('@expo/metro-runtime/error-overlay');
+            //     (0, expo_1.registerRootComponent)(withErrorOverlay(Component));
+            // }
+            // else {
                 (0, expo_1.registerRootComponent)(Component);
-            }
+            // }
         });
     }
     catch (e) {
